drivers/hmis/app/models/hmis/form/instance_enrollment_match.rb

```diff
@@ -6,6 +6,7 @@
 
 # determine if enrollment is a valid match for a form intance
 #
+require 'memery'
```

Comment:
for some reason we have 'require: false' in the gemfile and then require memery when needed. Not sure why...